addons/calendar/models/res_partner.py

```diff
@@ -22,7 +22,7 @@ def _compute_meeting_count(self):
             p.meeting_count = len(result.get(p.id, []))
 
     def _compute_meeting(self):
-        if self.ids:
+        if self.ids and self.exists():
```

Comment:
For a better readability you could just use `exists()`
```suggestion
        if self.exists():
```

see

https://github.com/odoo/odoo/blob/910756df630c52ac143a4d25a4acd06bf2131cd5/odoo/models.py#L5159-L5160

Reply:
Changes are done!